src/main/java/bio/terra/janitor/service/cleanup/FlightSubmissionFactoryImpl.java

```diff
@@ -22,6 +22,10 @@ public FlightSubmission createSubmission(TrackedResource trackedResource) {
         return FlightSubmission.create(GoogleBucketCleanupFlight.class, flightMap);
       case GOOGLE_BLOB:
         return FlightSubmission.create(GoogleBlobCleanupFlight.class, flightMap);
+      case GOOGLE_BIG_QUERY_DATASET:
```

Comment:
Thanks! Changed to BIGQUERY here.
And for the naming pattern. It would be CLOUDNAME_RESOURCENAME.
So I don't think we ever drop 'google' prefix anywhere. 
But for BLOB and BUCKET, we drop STORAGE because I think those two name are unique for GOOGLE.
But for Dataset and table, we should keep bigquery because those two word are not unique even in google. 
What we will have for now is:
GOOGLE_PROJECT, GOOGLE_BLOB, GOOGLE_BUCKET, GOOGLE_BIGQUERY_TABLE, GOOGLE_BIGQUERY_DATASET. 
does this sounds reasonable?

Reply:
👍 looks great